README.md

```diff
@@ -306,10 +306,18 @@ https://www.google.com/chrome/dev/
 
 #### 2. Build ChromeDriver BiDi
 
+Oneshot:
+
 ```sh
 npm run build
 ```
 
+Continuously with live-reload:
+
+```sh
+npm run watch
```

Comment:
Does it re-run the Mapper?

Reply:
No. But it's easy to accomplish so with `entr`. I'll remove "live-reload" to make it clearer.